src/electionguard/convert.cpp

```diff
@@ -0,0 +1,46 @@
+#include "date/date.h"
+
+#include <chrono>
+#include <codecvt>
+#include <iomanip>
+#include <iostream>
+#include <kremlin/lowstar_endianness.h>
+#include <sstream>
+#include <string>
+#include <vector>
+
+using std::string;
+using time_point = std::chrono::system_clock::time_point;
+using std::get_time;
+using std::gmtime;
+using std::mktime;
+using std::stringstream;
+using std::chrono::system_clock;
+
+namespace electionguard
+{
+    string timePointToIsoString(const time_point &time, const string &format)
```

Comment:
had to add the date library to get support for iso formats to work properly on windows.  it was causing issues when included in the header, particularly when compiling C, so it's moved into a code behind so the lib doesn't need to be included in downstream clients